localisation/00_ciwc_united_localisation.csv

```diff
@@ -17,4 +17,19 @@ great_sea_trade_route_tooltip;Can only build 
 ;;;;;;;;;;;;;;x
 text_effect_major_trade_route;This is �GMajor�! Trade Route;;;;;;;;;;;;;x
 text_effect_medium_trade_route;This is �YMedium�! Trade Route;;;;;;;;;;;;;x
-text_effect_minor_trade_route;This is �RMinor�! Trade Route;;;;;;;;;;;;;x
\ No newline at end of file
+text_effect_minor_trade_route;This is �RMinor�! Trade Route;;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+tp_road_tavern;Road Tavern ;;;;;;;;;;;;;x
+tp_zeppelin_tower;Zeppelin Tower ;;;;;;;;;;;;;x
+tp_resting_lounge;Resting Lounge ;;;;;;;;;;;;;x
+tp_jewelcrafters;Jewelcrafters;;;;;;;;;;;;;x
+tp_kodo_breeders;Kodo Breeders;;;;;;;;;;;;;x
+tp_turtle_pools;Turtle Pools;;;;;;;;;;;;;x
+tp_guide_eagle_nests;Guide Eagle Nests;;;;;;;;;;;;;x
+tp_wolf_bestiary;Wolf Bestiary;;;;;;;;;;;;;x
+tp_voodo_charm_vendor;Voodoo Charm Vendor;;;;;;;;;;;;;x
+tp_dwarfish_stonemasons;Dwarfish Stonemasons;;;;;;;;;;;;;x
+tp_chandlers_guild;Chandlers Guild;;;;;;;;;;;;;x
+tp_merchant_bots;Merchant Bots;;;;;;;;;;;;;x

```

Comment:
Are these actual robots? If not, `boats`.

Reply:
Its gnomish cultural so yes they are bots